src/redux/rootReducer.js

```diff
@@ -16,6 +16,7 @@ import myaccountReducer from './myaccountSlice';
 import planDetailsReducer from './planDetailsSlice';
 import paymentDetailsSliceReducer from './paymentDetailsSlice';
 import transactionsReducer from './transactionsSlice';
+import consentsReducer from './consentsSlice';
```

Comment:
as mentioned above I would prefer to name it 'publisherConsents' instead od 'consents'. In the future we will also more customer consents here, so would be good to have a clear names for both resources